PacketDotNet/IpPacket.cs

```diff
@@ -116,20 +116,9 @@ public override Packet PayloadPacket
 
         /// <value>
         /// The protocol of the ip packet's payload
-        /// Named 'Protocol' in IPv4
-        /// Named 'NextHeader' in IPv6'
         /// </value>
         public abstract IPProtocolType Protocol { get; set; }
 
-        /// <value>
-        /// The protocol of the ip packet's payload
-        /// Included along side Protocol for user convienence
-        /// </value>
-        public virtual IPProtocolType NextHeader
-        {
-            get => Protocol;
-            set => Protocol = value;
-        }
```

Comment:
For convenience and to not break existing programs, it would best be kept. I do think that it makes sense to use Protocol in most cases instead of NextHeader (similar to what you've done in IPv4Header).